packages/css-data/src/parse-css.test.ts

```diff
@@ -124,6 +158,44 @@ describe("Parse CSS", () => {
             },
           ],
         }
-      `);
+    `);
+  });
+
+  test("parse child combinator", () => {
+    expect(parseCss(`a > b { color: #ff0000 }`)).toMatchInlineSnapshot(`
+      {
+        "a>b": [
```

Comment:
what is the plan with these?

Reply:
alright lets skip those for now during generation

Reply:
done